kf_lib_data_ingest/etl/ingest_pipeline.py

```diff
@@ -126,6 +138,26 @@ def __init__(
             f'-- Ingest Params --\n{pformat(kwargs)}'
         )
 
+    def _validate_stages_to_run_str(self, stages_to_run_str):
+        """
+        Validate `stages_to_run_str`, a char sequence where each char
+        represents an ingest stage to run during pipeline execution
+
+        Each char must be in the set of valid chars defined by the keys of
+        CODE_TO_STAGE_MAP.
+
+        Order of chars does not matter because the pipeline always executes
+        stages (_iterate_stages) in the correct order.
+        """
+        assert_safe_type(stages_to_run_str, str)
```

Comment:
this would need to be in \_\_init\_\_ before the call to lower. If it's not a string, lower will error.